Technomancy/app.ts

```diff
@@ -4210,6 +4215,10 @@ window.onload = () => {
         }
     });
 
+    if ( buildings["s_manastone"].amount >= 200 ) {
+        $("#scratchpad").removeClass("hidden");
```

Comment:
This should probably in `window.onload` instead of in `load()`. `load()` is used to retrieve the gamestate, `window.onload` is for more visual stuff.